lms/static/sass/views/_verification.scss

```diff
@@ -165,7 +165,7 @@
 
   // elements - controls
   .action-primary {
-    @extend %btn-primary-blue;
+    @extend %btn-verify-primary;
```

Comment:
This will also make the "Verify Now" button shown on the receipt page green:

![](https://i.imgur.com/oJJwqxV.png)

I don't think that's a problem, but it's not what was asked for as part of this work.